tools/ansible/tasks/wsu/main.yaml

```diff
@@ -191,16 +206,42 @@
             src: "{{ tmp_dir.path }}/kubernetes/node/bin/kube-proxy.exe"
             dest: "{{ tmp_dir.path }}/kube-proxy.exe"
 
+        - name: Download checksum file for cni plugins
+          get_url:
+            url: "{{ cni_plugins_location }}.sha512"
```

Comment:
Why should we download the checksum url and then do integrity check? Why can we use checksum directly on url like this:
`sha256:http://example.com/path/sha256sum.txt` given here https://docs.ansible.com/ansible/latest/modules/get_url_module.html#parameter-checksum

Reply:
I did not know about this. Fixed it